core/templates/pages/donate-page/donate-page.scripts.ts

```diff
@@ -23,4 +23,4 @@ require('App.ts');
 
 require('pages/OppiaFooterDirective.ts');
 
-require('pages/donate-page/donate-page.controller.ts');
+require('pages/donate-page/donate-page.component.ts');
```

Comment:
Also I think, as we are importing the component in the `donate-page.module.ts` we don't need this.

Reply:
Done